integrated_channels/cornerstone/views.py

```diff
@@ -163,16 +156,4 @@ def get(self, request, *args, **kwargs):
                                 f'fixing modified/header mismatch')
                     if_modified_since_dt = datetime.datetime.fromtimestamp(if_modified_since)
                     item['LastModifiedUTC'] = if_modified_since_dt.strftime(ISO_8601_DATE_FORMAT)
-        duration_seconds = time.time() - start_time
-        headers_dict = dict(request.headers)
-        headers_json = json.dumps(headers_dict)
-        IntegratedChannelAPIRequestLogs.store_api_call(
-            enterprise_customer=enterprise_customer,
-            enterprise_customer_configuration_id=enterprise_config.id,
-            endpoint=request.get_full_path(),
-            payload=f"Request Headers: {headers_json}",
-            time_taken=duration_seconds,
-            status_code=200,
-            response_body=json.dumps(data)
-        )
```

Comment:
🤔

Reply:
while pulling changes i was only keeping the changes from your branch that were according to the scope of my PR
that is why this was getting removed
updated.